CRM/Activity/BAO/Activity.php

```diff
@@ -2071,16 +2071,10 @@ public static function addActivity(
     $targetContactID = NULL,
     $params = array()
   ) {
-    $date = date('YmdHis');
+    $date = CRM_Utils_Date::currentDBDate();
```

Comment:
Can we keep this out of this PR & address it separately - it could potentially create a change which is not the intent of this PR

Reply:
@eileenmcnaughton I've removed this from the PR.  It doesn't impact this change anyway but I had thought they were equivalent, thought now realise they may not be.